### DISCUSSION TITLE
Pinkish image becoming bluish

### DISCUSSION THREAD

**Discussion by khirulsp** [dlemstra/Magick.NET#1191] [Help]
I have a pdf to image conversion using the following code:

  ```
var settings = new MagickReadSettings();
  settings.Density = new Density(96);
  settings.ColorSpace = ColorSpace.CMYK;
  settings.Compression = CompressionMethod.Group4;

  using (var images = new MagickImage(inputPdfPath, settings))
  {
      image.Format = MagickFormat.Jpeg;
      images.Write(tmpOutputFilePath);
  }
```

The image produced is as follows as I expected it:
![image](https://user-images.githubusercontent.com/90170080/171855966-001a91eb-b081-409a-9e35-25e83cff937b.png)

But when I show the image in browser html it shows as below color:
![image](https://user-images.githubusercontent.com/90170080/171855708-e9e32c48-b110-414a-b497-2476e5640c5b.png)

Can anyone suggest what I am doing wrong here?

**Comment by dlemstra** [comment]
Can you share the pdf file that was used?

**Reply by khirulsp** [reply]
I actually could resolve the issue as following
```
var settings = new MagickReadSettings();
settings.Density = new Density(96);
settings.ColorSpace = ColorSpace.CMYK;

using (var image = new MagickImage(inputPdfPath, settings))
{
    image.Format = MagickFormat.Jpeg;
    image.TransformColorSpace(ColorProfile.CoatedFOGRA39, ColorProfile.SRGB); // this color profile fixed it
    image.Strip();
    image.Write(tmpOutputFilePath);
}
```
